src/Apps/Artist/routes.tsx

```diff
@@ -69,8 +69,13 @@ export const routes: RouteConfig[] = [
         prepareVariables: (params, props) => {
           // FIXME: The initial render includes `location` in props, but subsequent
           // renders (such as tabbing back to this route in your browser) will not.
-          const initialFilterState = props.location ? props.location.query : {}
-          return { ...initialFilterState, ...params }
+          const filterStateFromUrl = props.location ? props.location.query : {}
+          const filterParams = Object.assign(
+            { ...filterStateFromUrl },
+            { ...params },
+            initialState
+          )
```

Comment:
```js
const filterParams = {
 ...filterStateFromUrl,
 ...params,
 ...initialState
}
```
Seems weird to mix object.assign and a spread, and also to spread just a single object into itself 

```
{ ...filterStateFromUrl }
```

Reply:
yea good call, this is weird

Reply:
Updated to the
```
const filterParams = {
 ...filterStateFromUrl,
 ...params,
 ...initialState
}
``` style.

With spreading you don't need `defaults` and `assign`, etc. as I'm used to.

(but, you do need to do it in reverse order from the order you would when using one of the above methods).

Reply:
This is a pretty common pattern with spreads: 

```js
const config = {
  pageNumberDefault: 10,
  someOtherThing: "hi!",
  ...overrides
}
```